### PR TITLE
signout workflow

### PR DESCRIPTION
**Pull request by edwinchiboy**
We worked on the signout workflow

### CODE REVIEW

**Comment by Blazebrain** [comment]
Fix Conflicts

**Comment by YenumJay** [comment]
Pull latest changes from dev and resolve conflicts.

File: lib/ui/view/sign_out/sign_out_view.dart [hunk 1]
```diff
@@ -0,0 +1,160 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import '../../../../general_widgets/custom_text.dart';
+import 'package:flutter/foundation.dart';
```
Comment (pauleke65) [inline lib/ui/view/sign_out/sign_out_view.dart:4]: Why is foundation being imported?

File: lib/ui/view/sign_out/sign_out_view.dart [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+              ],
+            ),
+          )
+        ],
+      ),
+    );
+  }
+}
+
+class BottomsheetContent extends StatelessWidget {
```
Comment (pauleke65) [inline lib/ui/view/sign_out/sign_out_view.dart:78]: Refactor to use bottom sheet service

File: lib/ui/view/sign_out/sign_out_view.dart [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+              Icons.logout_sharp,
+              color: AppColors.zuriPrimaryColor,
+            ),
+          ),
+        )
+      ],
+    );
+  }
+
+  Future<void> _showSignOutDialog(context) async {
```
Comment (pauleke65) [inline lib/ui/view/sign_out/sign_out_view.dart:113]: Refactor to use Dialog Service

File: lib/ui/view/workspace/workspace_view/workspace_view.dart [hunk 4]
```diff
@@ -1,11 +1,16 @@
 import 'package:flutter/material.dart';
+import 'package:flutter/foundation.dart';
```
Comment (pauleke65) [inline lib/ui/view/workspace/workspace_view/workspace_view.dart:2]: Foundation again?

File: lib/ui/view/workspace/workspace_view/workspace_view.dart [hunk 5]
```diff
@@ -33,7 +38,28 @@ class WorkspaceView extends StatelessWidget {
...
+                                mainAxisAlignment:
+                                    MainAxisAlignment.spaceBetween,
+                                children: [
+                                  WorkSpaceDisplayInfo(
+                                      'assets/logo/zuri_chat_logo.png',
+                                      'Hngi8 x i4G',
+                                      'hngi8.slack.com'),
+                                  IconButton(
+                                      onPressed: () {
+                                        showModalBottomSheet(
```
Comment (pauleke65) [inline lib/ui/view/workspace/workspace_view/workspace_view.dart:52]: Add the bottom sheet service and the dialog from the model, which should use their respective services

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Changes Requested

**Comment by pauleke65** [comment]
Similar PR available [#828 ](https://github.com/zurichat/zc_app/pull/828)

**Pull request closed without merging**